Source/Core/Common/StringUtil.cpp

```diff
@@ -559,12 +567,12 @@ std::string UTF16ToUTF8(const std::wstring& input)
   return CodeToUTF8("UTF-16LE", input);
 }
 
-#endif
-
 std::string UTF16BEToUTF8(const char16_t* str, size_t max_size)
 {
   const char16_t* str_end = std::find(str, str + max_size, '\0');
-  std::wstring result(static_cast<size_t>(str_end - str), '\0');
+  std::u16string result(static_cast<size_t>(str_end - str), '\0');
   std::transform(str, str_end, result.begin(), static_cast<u16 (&)(u16)>(Common::swap16));
-  return UTF16ToUTF8(result);
+  return CodeToUTF8("UTF-16LE", result);
```

Comment:
I think it should be possible to remove the `std::transform` line if you replace `"UTF-16LE"` with `"UTF-16BE"`.

Reply:
See commit a32d38817f6000594a757de683f5835b01073387.